docs/pages/project/contributing/contributing-ui.md

```diff
@@ -75,6 +75,14 @@ The designs in this specification should result in enabling:
 
 ## Setup
 
+### Node Version Recommendations 
+We recommend to use LTS versions of node:
+
+- Node 16 LTS: https://nodejs.org/download/release/v16.19.0/ \
```

Comment:
Does Meshery support both? Let's reduce to one unless there is a requirement or old dependency that needs Node 16.

Reply:
Yes, it does both. because it does both, its written here. Workflow files use the node v16 for building the meshery-ui; and contributors/maintainers are using node18. nvmrc suggests node18 too.

before you question why node16 in ci, we have set the secret to 18.12.1, but still implicitly it uses node16, we haven't found the reason for this behaviour yet.

Reply:
Use of Node 16 might very well be happening based on what specific GitHub Actions are using under the covers in our workflows.